.github/workflows/sync-packgejson-and-changelog-after-auto-release.yml

```diff
@@ -0,0 +1,53 @@
+name: 🔁 Sync Package.json and CHANGELOG After Auto Release
+
+# Do it after auto release
+on:
+  workflow_run:
+    workflows: ["🤖 Auto Release"]
+    branches:
+      - stable
+      - alpha # just for test semantic-release
+    types:
+      - completed
+
+jobs:
+  sync-site-lock:
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.conclusion == 'success'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: 'latest'
+
+      - name: Use Node.js 14.17
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.17'
+          cache: 'npm'
+
+      - name: Git bootstrap
+        run: |
+          git config --global user.name 'pddpd'
+          git config --global user.email 'pddpengdi@gmail.com'
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/$GITHUB_REPOSITORY
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+
+      - name: Checkout branch
+        run: |
+          git checkout -b chore-sync
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Push
+        run: |
+          git add .
+          git commit -m "chore: 🤖 update package.json and CHANGELOG" --no-verify
```

Comment:
hmmm...personally, I don't like the emoji in git commits...

Reply:
done